src/components/reference/_triage-states.mdx

```diff
@@ -17,6 +17,9 @@ Findings can also be **removed**. Semgrep considers a finding removed if it is n
 
 Your removed findings do not count toward the fix rate or the number of findings. The removed findings also do not appear in Semgrep AppSec Platform.
 
-:::info Findings triaged across refs
-Findings triaged (ignored, reopened) in a specific branch, PR, or MR are also triaged in all other branches, PRs, and MRs of a particular repository. Additionally, if you filter for [Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References) (refs) on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, then triage a finding, the finding is also automatically triaged in all other branches, PRs, MRs, and refs.
-:::
+### Triage behavior across refs and branches
+
+- Findings that are triaged as ignored or reopened in a specific branch, PR, or MR are **not** automatically triaged in all other branches, PRs, and MRs of a particular repository.
+- If you filter for [Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References) (refs) on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, then triage a finding, the finding is **NOT** automatically triaged in all other branches, PRs, MRs, and refs.
+    - Triaging a finding on the default branch **does** affect all related non-default refs or branch-specific refs with an instance of the finding **when the finding was triaged**.
+- Future instances of a finding found at a later date aren't automatically triaged, even if the finding on the default has been triaged.
```

Comment:
Thanks for writing this up! I think this doesn't quite describe 100% of the behavior. We triage across refs in all instances except one. Clara describes that specific instance pretty well in [our internal docs](https://www.notion.so/semgrep/Triage-Across-Refs-4dacd1e5ea814801bf6e06186790d2e0?pvs=4#4518acb6d78a41a68c34805e9bb788a5) but it is when a finding is newly introduced in a diff scan, it is **not** automatically triaged to ignore at scan time even if there have been other instances on other branches that have been triaged to ignore.